src/borg/testsuite/__init__.py

```diff
@@ -147,8 +147,8 @@ def is_utime_fully_supported():
         else:
             open(filepath, "w").close()
         try:
-            os.utime(filepath, (1000, 2000), follow_symlinks=False)
-            new_stats = os.stat(filepath, follow_symlinks=False)
+            os.utime(filepath, (1000, 2000), follow_symlinks=False if os.utime in os.supports_follow_symlinks else True)
+            new_stats = os.stat(filepath, follow_symlinks=False if os.stat in os.supports_follow_symlinks else True)
```

Comment:
Hmm, guess that needs to be done differently:
The code right above either creates a symlink (with a non-existing target as it seems) or a file.
So we can't just arbitrarily switch follow_symlinks to True/False below:
- the target does not exist
- also, line 144 seems to imply we want to know about utime behaviour **on symlinks**

So guess this needs double-checking:
- what tests do use this and what exactly do we need to check here?
- the utime call needs to match the fs object it is used with

Reply:
Also: never use a condition to decide between True or False. Above, you could've just used `follow_symlinks=os.stat not in os.supports_follow_symlinks`.

Reads a bit strange, but guess that's because the default is with True and the special (and sometimes unsupported) behaviour is with False.